Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -136,7 +137,7 @@ async function run() {
         tl.setResult(tl.TaskResult.Failed, err.message);
     }
     finally {
-        if (isKubConfigLogoutRequired(command) || externalAuth) {
+        if (isKubConfigLogoutRequired(command)) {
```

Comment:
Why not adding this:
`isKubConfigLogoutRequired(command) || !externalAuth`

In case of scenarios like `helm get` command, isKubConfigRequired=false, externalAuth=false, so `kubectlCli` will not be initialized in the first place. If we add !externalAuth here, kubectlCli.logout() will be returning `undefined`.